description/client/components/App.jsx

```diff
@@ -27,8 +30,11 @@ export default class App extends React.Component {
         <h1>{this.state.currentCourse.title}</h1>
         <h2>{this.state.currentCourse.headline}</h2>
         <Feedback currentCourse={this.state.currentCourse} />
-        <h4>{this.state.currentCourse.name}</h4>
-        <h4>{new Date(this.state.currentCourse.created).toDateString()}</h4>
+        <h4>Created by {this.state.currentCourse.name}</h4>
+        <h4>
+          Last Updated{' '}
```

Comment:
Also did this sort of thing trying to format a string. Curious to see if there's a best practices sort of thing for formatting this